src/main/java/org/jenkinsci/plugins/EnvironmentVarSetter.java

```diff
@@ -6,10 +6,10 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import edu.umd.cs.findbugs.annotations.CheckForNull;
```

Comment:
does not look like pure java or jenkins dependencies

Reply:
SpotBugs annotations are included by default and the [recommended replacement](https://www.jenkins.io/doc/developer/tutorial-improve/replace-jsr-305-annotations/) for JSR 305

Reply:
I moved this out to PR #82